### ISSUE TITLE
How to Retrieve Full Size (Not Resized/Thumbnail) Image Datatype

### ISSUE THREAD

**Issue by dibyanugraha** [microsoft/PowerApps-Samples#194]
As mentioned in here @[Documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/sample-set-retrieve-entity-images), and the sample, retrieving using RetrieveMultiple does not return the full size image, instead it returns the resized/thumbnail version of the image. The sample application @[SetRetrieveImages Sample Link](https://github.com/microsoft/PowerApps-Samples/tree/master/cds/orgsvc/C#/SetRetrieveImages) does not provide on how to retrieve the full size image correctly.

**Comment by phecke** [comment]
@dibyanugraha According to the documentation, when using Retrieve or RetrieveMultiple, you must explicitly specify the image column and not ask for AllColumns to get the full sized image.

There is also updated image documentation here: https://docs.microsoft.com/en-us/powerapps/developer/data-platform/image-attributes

**Comment by dibyanugraha** [comment]
"Retrieve or RetrieveMultiple, you must explicitly specify the image column and not ask for AllColumns to get the full sized image."
It's true -- I don't use the AllColumns = true and I have explicitly written in the my FetchXML which image column that I need to retrieve. The image is retrieved but in thumbnail size instead of full size.

The updated image documentation however does not mention anything about using RetrieveMultiple. It resorts to consume web API which is quite lengthy process to setup just to receive full image size instead of just calling FetchMultiple using FetchXML/QueryExpression.

In short: Is there any way to mimic this
![image](https://user-images.githubusercontent.com/11976855/125732103-b7a45e50-5908-425b-b984-d30cdd693e22.png)
Into a "friendly" FetchXML or QueryExpression?
